.github/workflows/R-CMD-check.yaml

```diff
@@ -71,11 +57,10 @@ jobs:
           needs: check
 
       - name: Install cmdstan
-        run: |
-          print(list.files("C:/rtools43"))
-          cmdstanr::check_cmdstan_toolchain(fix = TRUE)
-          cmdstanr::install_cmdstan(cores = 2, quiet = TRUE)
-        shell: Rscript {0}
+        uses: epinowcast/actions/install-cmdstan@v1
```

Comment:
This is a better version of installing `cmdstanr` in some way?

Reply:
> (mostly this is using our cmdstan action which has caching to speed up builds of CI)

So if a version of `cmdstanr` is available installed {somewhere} already then it won't try to reinstall it? Where would the version be stored?

Reply:
No this is using `cmdstanr` to install and cache `cmdstan` - it gets cached using the github actions caching system which is also what is happening to R package dependencies